assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
> Trim is to ensure the following check always works with hash as a first character in the URL

I think its an agreeing to disagree on this. If `.trim()` is needed for a use case where extra spaces are in the markup due to the way the scala is written, then no probs lets leave that in there 👍  
IMO feels a bit of an edge case seeing this with `hrefs`